airbyte-integrations/connectors/source-chargebee/source_chargebee/schemas/event.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "name": "Item",
+  "name": "Event",
   "type": "object",
```

Comment:
Is this a breaking change?  The stream name has changed maybe?

Reply:
No, this is not breaking change, since the schema types are in place.